EIPS/eip-7547.md

```diff
@@ -126,7 +126,7 @@ class BeaconBlockBody(Container):
 
 **Builder and API spec:**
```

Comment:
Feels odd for the EIP to reference a builder API, given that it is an element outside of the protocol. As I'm not well-versed in the EIP process, is it common for such external to be mentioned? 

Additionally, regarding the endpoint modification, builder could just monitor the inclusion list through the gossip network so no additional modification of end point is required.

Reply:
Okay; I've updated the PR to remove the offending entry.